src/main/java/org/alliancegenome/curation_api/base/dao/BaseSQLDAO.java

```diff
@@ -295,8 +301,9 @@ public SearchResponse<E> searchByParams(Pagination pagination, Map<String, Objec
                                     });
                                 });
                             }
-                        }
-                    }); 
+									if(StringUtils.isNotEmpty(notNullElement)) {
+                               b.must(m -> m.bool(s -> s.should(p.exists().field(notNullElement))));}
```

Comment:
@cmpich instead of `notNullElement` can we call this `nonNullField`?

Reply:
Sure.